### DISCUSSION TITLE
How do I create a Docker image correctly, including the necessary files?

### DISCUSSION THREAD

**Discussion by allaga** [squidfunk/mkdocs-material#7577] [Q&A]
Hello!

First of all, I would like to express my gratitude for such a great product as Material for Mkdocs. I am just starting my journey in the IT field and I have thoroughly researched Mkdocs and the Material for Mkdocs documentation. However, as a beginner, I have encountered some difficulties with Docker.

How do I correctly create a Docker image, including the necessary files, the "Overrides" folder, and the "mkdocs.yml" file?

Thank you for your help!

**Comment by alexvoss** [comment]
Hi, that is a very broad question. What is it that you are trying to achieve that the [official docker image](https://squidfunk.github.io/mkdocs-material/getting-started#with-docker) does not do? 

If you really need to brew your own image, have you tried looking for tutorials or examples?
